lib/s3routes/routes/routeGET.js

```diff
@@ -74,6 +74,7 @@ function routerGET(request, response, api, log, statsClient, dataRetrievalFn) {
                 });
         }
     } else {
+        /*eslint-disable no-lonely-if*/
```

Comment:
Why disable it here?

Reply:
I wasn't sure how to refactor this part because each nested if seemed like it served a purpose :/